pkg/daemon/pivot/utils/run.go

```diff
@@ -48,9 +49,11 @@ func runExtBackoff(backoff wait.Backoff, command string, args ...string) (string
 		return true, nil
 	}); err != nil {
 		if err == wait.ErrWaitTimeout {
-			return "", errors.Wrapf(lastErr, "failed to run command %s (%d tries): %v", command, backoff.Steps, err)
+			errs := kubeErrs.NewAggregate([]error{err, lastErr})
+			return "", fmt.Errorf("failed to run command %s (%d tries): %w", command, backoff.Steps, errs)
 		}
-		return "", errors.Wrap(err, "failed to run command %s (%d tries): %v")
+		errs := kubeErrs.NewAggregate([]error{err, err})
```

Comment:
Shouldn't this be `errs := kubeErrs.NewAggregate([]error{err, lastErr})`?

Reply:
@cheesesashimi Hmm, looking back at the original code, seems like the author was not wrapping `lastErr` here. Should we still wrap the last error here or should the original line of code be replaced with something like:
```
return "", fmt.Errorf("failed to run command %s (%d tries): %w", err)
```

Reply:
I don't think it matters too much in this case. Let's go with what was there before.